qa/workunits/rbd/rbd_mirror_bootstrap.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh -ex
+#
+# rbd_mirror_bootstrap.sh - test peer bootstrap create/import
+#
+
+. $(dirname $0)/rbd_mirror_helpers.sh
+
+remove_peer() {
+    local cluster=$1
+    local pool=$2
+
+    local uuid=$(rbd --cluster ${cluster} --pool ${pool} mirror pool info --format xml |
+        ${XMLSTARLET} sel -t -v  '//mirror/peers/peer[1]/uuid')
```

Comment:
nit: there is a peculiarity with `local` that if you use it like here, i.e.: 
```
local var=$(cmd)
```
and the `cmd` fails for some reason, it will not abort (as one could expect due to `-e` flag set). If you want to abort if the `cmd` fails, you have to use:
```
local var
var=$(cmd)
```